kms/attestations/verify_attestation.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to verify HSM attestations using
+certificate bundle obtained from Cloud HSM.
+
+For more information, visit https://cloud.google.com/kms/docs/attest-key.
+"""
+
+# [START verify_attestations]
+import argparse
+from cryptography import x509
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat import backends
+from cryptography import exceptions
+import gzip
+import pem
+
+
+def get_attestation_contents(attestation):
+    """Gets the contents of an attestation, and uncompresses it if necessary.
```

Comment:
Gets the attestation statement. If the attestation statement is compressed,  uncompresses it automatically.